pkg/controller/migmigration/task.go

```diff
@@ -1337,12 +1337,19 @@ func (t *Task) getDestinationClient() (compat.Client, error) {
 }
 
 // Get the persistent volumes included in the plan which are not skipped.
-func (t *Task) getPVs() migapi.PersistentVolumes {
+// This function will only return PVs that are being copied via restic or snapshot
+func (t *Task) getStagePVs() migapi.PersistentVolumes {
```

Comment:
can we name this `getResticOrSnapshotPVs`? sorry but took me a while to figure out `Stage` means `ResticOrSnapshot`and then saw it in the comments

Reply:
@alaypatel07 this includes `Move` PVs as well which is why I included `Stage` in the name. I.e. any PV that needs to be included in a stage backup.

What do you think?

Reply:
I'll add a comment